.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+pnpm test
+
+pnpm prettier
```

Comment:
Now that this is pre-push, might make sense to make this a `lint` step rather than `prettier`. Running `prettier` writes files, so you may end up with not-yet-staged/committed/pushed changes after this runs.

Reply:
@holic So true! I should have caught that 🤦